docs/mp/guides/39_jbatch.adoc

```diff
@@ -0,0 +1,329 @@
+
+<2> The second step of the job includes `MyBatchlet`.
+
+NOTE: You must specify the fully qualified names in the `ref` properties, like “io.helidon.jbatch.example.jobs.MyItemReader”, otherwise it will not work.
+
+== Create an Endpoint
+Create a small endpoint to activate the job:
+
+[source,java]
+.new endpoint
+
+----
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public JsonObject executeBatch() throws Exception{
+
+        BatchSPIManager batchSPIManager = BatchSPIManager.getInstance();
+        batchSPIManager.registerPlatformMode(BatchSPIManager.PlatformMode.SE);
+        batchSPIManager.registerExecutorServiceProvider(new HelidonExecutorServiceProvider());
+
+        JobOperator jobOperator = getJobOperator();
+        Long executionId = jobOperator.start("myJob", new Properties());
+        Thread.sleep(3000);
+        JobExecution jobExecution = jobOperator.getJobExecution(executionId);
+
+
+        List<StepExecution> stepExecutions = jobOperator.getStepExecutions(executionId);
+        List<String> executedSteps = new ArrayList<>();
+        for (StepExecution stepExecution : stepExecutions) {
+            executedSteps.add(stepExecution.getStepName());
+        }
+
+
+        return JSON.createObjectBuilder()
+                .add("Steps executed", Arrays.toString(executedSteps.toArray()))
+                .add("Status", jobExecution.getBatchStatus().toString())
+                .build();
+    }
+}
+----
+
+Helidon specifies to JBatch that it should run in Standalone (SE) mode. It will also register the `HelidonExecutorServiceProvider` which is actually relatively small. For our example we need something really small, like a `FixedTheadPool` with 2 threads. This provider is used to tell our JBatch engine exactly which ExecutorSevice to use.
+
+[source, xml]
+.HelidonExecutorServiceProvider
+
+----
+public class HelidonExecutorServiceProvider implements
+ExecutorServiceProvider {
+    @Override
+    public ExecutorService getExecutorService() {
+        return Executors.newFixedThreadPool(2);
+    }
+}
+----
+
+== Run the Code 
+
+[source,bash]
+
+----
+mvn package
+java -jar target/helidon-jbatch-example.jar 
+----
+
+
+== Call the Endpoint
+
+[source,bash]
+
+----
+curl -X GET http://localhost:8080/batch
+----
+
+You should receive the following log:
+
+[source,bash]
+
+----
+processItem: MyInputRecord: 1
+processItem: MyInputRecord: 2
+processItem: MyInputRecord: 3
+writeItems: [MyOutputRecord: 2, MyOutputRecord: 6]
+processItem: MyInputRecord: 4
+processItem: MyInputRecord: 5
+processItem: MyInputRecord: 6
+writeItems: [MyOutputRecord: 10]
+processItem: MyInputRecord: 7
+processItem: MyInputRecord: 8
+processItem: MyInputRecord: 9
+writeItems: [MyOutputRecord: 14, MyOutputRecord: 18]
+processItem: MyInputRecord: 10
+Running inside a batchlet
+----
+
+and the following result:
+
+[source,bash]
+
+----
+{"Steps executed":"[step1, step2]","Status":"COMPLETED"}
+----
+
```

Comment:
Please add the following:

To check job status, simply call:

[source,bash]
----
 curl -X GET http://localhost:8080/batch/status/1 (1)
----
(1) Job Id may differ.

and you should see the following output:

[source,bash]
----
{"Steps executed":"[step1, step2]","Status":"COMPLETED"}
----